tests/folium_tests.py

```diff
@@ -58,7 +58,8 @@ def test_init(self):
         assert self.map.location == [45.5236, -122.6750]
         assert self.map.map_size == {'width': 900, 'height': 400}
 
-        nt.assert_raises(ValueError, callableObj=folium.Map)
+        # TODO: assert that does not raise?
+        #nt.assert_raises(ValueError, callableObj=folium.Map)
```

Comment:
We do not need to test for this.  Lets just add a twin test `test_manual_bounds`.